experimental/dds/sequence-deprecated/src/test/sparseMatrix.spec.ts

```diff
@@ -14,7 +14,12 @@ import {
 	MockStorage,
 } from "@fluidframework/test-runtime-utils/internal";
 
-import { SparseMatrix, SparseMatrixFactory, SparseMatrixItem } from "../sparsematrix.js";
+import {
+	SparseMatrix,
```

Comment:
Is this needed if we now have `SparseMatrixClass`?

Reply:
SparseMatrixClass could be used instead as a type (currently they are the same since sparse matrix does not define an interface like most of our shared object, which is required for it to avoid leaking its class (and thus SharedObject) into the API. Since its only `@internal` that seems fine.

Anyway, looking at the test usage, migrating this to use SparseMatrixClass type would produce a larger diff for no real gain, but I did see it could use the newer factory pattern to create instance in the test via SparseMatrix.create, so I updated it to use that.